src/debugger/sourceMapsCombinator.ts

```diff
@@ -90,10 +90,21 @@ export class SourceMapsCombinator {
     }
 
     private readSourcemap(file: string, code: string): SourceMapConsumer | null {
-        let result = sourceMapResolve.resolveSync(code, file, fs.readFileSync);
+        let result = sourceMapResolve.resolveSync(code, file, readFileSync.bind(null, getDiskLetter(file)));
         if (result === null) {
             return null;
         }
         return new SourceMapConsumer(result.map);
     }
 }
+
+// Hack for source-map-resolve and cutted disk letter
+// https://github.com/lydell/source-map-resolve/issues/9
+function readFileSync(diskLetter: string, filePath: string) {
+    return fs.readFileSync(`${diskLetter}${filePath}`);
```

Comment:
Yes, now it works that way, but I can imagine that when `urix` transitive dependency (which has the bug we're trying to workaround) gets updated we'd run into trouble which we could avoid by adding just one additional check